.github/workflows/main.yaml

```diff
@@ -290,5 +289,6 @@ jobs:
         run: |
           cargo clippy
           cargo fmt --check
-          cargo run --example=example
-          cargo test --verbose
+          cargo run --example=example --features compile-stan-model
+          cargo test --verbose --features compile-stan-model
+          cargo test --verbose --features compile-stan-model model_compiling -- --ignored
```

Comment:
I think we should test without the compile feature as well, just to make sure the `if cfg` are all correct.

Reply:
done